flang/lib/Lower/IntrinsicCall.cpp

```diff
@@ -425,11 +426,16 @@ struct IntrinsicLibrary {
   template <Extremum, ExtremumBehavior>
   mlir::Value genExtremum(mlir::Type, llvm::ArrayRef<mlir::Value>);
   mlir::Value genFloor(mlir::Type, llvm::ArrayRef<mlir::Value>);
-  mlir::Value genIAnd(mlir::Type, llvm::ArrayRef<mlir::Value>);
+  mlir::Value genIand(mlir::Type, llvm::ArrayRef<mlir::Value>);
+  mlir::Value genIbclr(mlir::Type, llvm::ArrayRef<mlir::Value>);
+  mlir::Value genIbits(mlir::Type, llvm::ArrayRef<mlir::Value>);
+  mlir::Value genIbset(mlir::Type, llvm::ArrayRef<mlir::Value>);
   fir::ExtendedValue genIchar(mlir::Type, llvm::ArrayRef<fir::ExtendedValue>);
   mlir::Value genIEOr(mlir::Type, llvm::ArrayRef<mlir::Value>);
   fir::ExtendedValue genIndex(mlir::Type, llvm::ArrayRef<fir::ExtendedValue>);
   mlir::Value genIOr(mlir::Type, llvm::ArrayRef<mlir::Value>);
```

Comment:
I don't really get our camelCaseNames here. Here, we're changing `IAnd` to `Iand`, while leaving `IEOr` and `IOr`. Why?

(The latter seems to more closely follow the MLIR Op naming style.)

Reply:
I didn't notice the latter two cases. (I thought I checked for more cases, but apparently not.)  I'll try to keep this in mind for a later update.

Reply:
Does that mean the naming scheme here is `gen` + Fortran name of intrinsic with only the first letter uppercase?

If that is the preferred naming convention, it should be spelled out explicitly in a comment above all these declarations. We'll want everyone, including contributors,  rowing in the same direction.